src/lucky/error_handler.cr

```diff
@@ -13,20 +13,10 @@ class Lucky::ErrorHandler
   def call(context : HTTP::Server::Context)
     call_next(context)
   rescue error : Exception
-    if settings.show_debug_output
-      print_debug_output(context, error)
-    else
-      call_error_action(context, error)
-    end
+    call_error_action(context, error)
   end
 
-  private def print_debug_output(context : HTTP::Server::Context, error : Exception) : HTTP::Server::Context
-    error.inspect_with_backtrace(error_io)
-    render_exception_page(context, error)
-    context
-  end
-
-  private def render_exception_page(context, error)
+  def self.render_exception_page(context, error)
     context.response.reset
     context.response.status_code = 500
     context.response.content_type = "text/html"
```

Comment:
On the line right below this, instead of printing right away, can you change this to return a Lucky::TextResponse? That way it can be rendered in a Lucky::Action

https://github.com/luckyframework/lucky/blob/1a16bebd932195118d48369aa2a22bb185f36dac/src/lucky/text_response.cr

```crystal
Lucky::TextResponse.new(
  context: context,
  content_type: "text/html",
  status_code: 500,
  body: Lucky::ExceptionPage.for_runtime_exception(context, error)
)
```